modules/gapi/include/opencv2/gapi/core.hpp

```diff
@@ -1732,6 +1732,7 @@ GAPI_EXPORTS GMat warpAffine(const GMat& src, const Mat& M, const Size& dsize, i
                              int borderMode = cv::BORDER_CONSTANT, const Scalar& borderValue = Scalar());
 //! @} gapi_transform
 
+namespace streaming {
 /** @brief Gets dimensions from Mat.
 
 @note Function textual ID is "org.opencv.core.size"
```

Comment:
Do you really need namespace here?

namespace and ID doesn't look consistent.

Reply:
I will fix the IDs thanks.